### PR TITLE
conformance tests: dump logs for all control plane pods

### PR DESCRIPTION
**Pull request by kdomanski**
**Why we need it**:
I'm debugging a failed test and according to the log, the kubermatic api server returns an empty error response. I'd like to be able to easily see what it has to say in its logs.

In addition, the existing code that dumps the controller-manager logs assumes that there is only one replica. I generalized it to always iterate over all pods, to avoid misleading the user.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: api/hack/ci/ci-run-minimal-conformance-tests.sh [hunk 1]
```diff
@@ -68,7 +68,18 @@ function cleanup {
     fi
 
     # Controller manager logs
-    kubectl logs -n $NAMESPACE  $(kubectl get pod -n $NAMESPACE -l role=controller-manager |tail -n 1|awk '{print $1}')
+    echodate "Dumping controller-manager logs"
+    for pod in $(kubectl get pods -n $NAMESPACE -l role=controller-manager -o go-template='{{ range .items }}{{.metadata.name}}{{ "\n" }}{{end}}'); do
```
Comment (alvaroaleman) [inline api/hack/ci/ci-run-minimal-conformance-tests.sh]: What about just dumping all logs of all pods in the ns?

Comment (kdomanski) [reply api/hack/ci/ci-run-minimal-conformance-tests.sh]: Sure, even better.

**Pull request merged**